advocacy_docs/partner_docs/CommVaultGuide/05-UsingCommvault.mdx

```diff
@@ -0,0 +1,233 @@
+---
+title: 'Using Commvault Express'
+description: 'Walkthroughs of multiple Commvault Express usage scenarios'
+---
+
+This section shows how backups are taken and restored from EDB Postgres Advanced Server (EPAS) using Commvault Express.
+
+## Backup and Restore using Commvault Express
```

Comment:
Title should be 
"DumpBasedBackupSet Backup and Restore"
Tell them about what it is and us this info as well "pg_dump Utility is used to take the backup."

Then you can go to "Taking DumpBasedbackupset backup" with steps.

So I am really moving the info around you have from line 8 to 15.

So section should look something like this.

Using Commvault Express
 
Commvault provides the two methods of taking the backup from EDB Postgres Advanced Server (EPAS) and restoring it.
DumpBasedBackupSet Backup and Restore
FSBasedBackupSet Backup and Restore
DumpBasedBackupSet Backup and Restore
Dump based backup uses the pg_dump Utility to take the backup.

Taking DumpBasedBackupSet Backup
	Steps for getting backup.
Restoring DumpBasedBackupSet Backup
Steps for restore 
FSBasedBackupSet Backup and Restore]
	At this time there is a known issue with FSBased Restore that does not allow for proper restoration of the database.  See the known issues section of the guide for more information.